intelmq/bots/parsers/malc0de/parser.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+""" IntelMQ parser for Malc0de feeds """
+
+import sys
+import dateutil.parser
+
+from intelmq.lib.bot import ParserBot
+from intelmq.lib.message import Event
+
+
+class Malc0deParserBot(ParserBot):
+
+    WINDOWS_FORMAT = {'http://malc0de.com/bl/BOOT',
+                      'http://malc0de.com/bl/BOOT'}
+
+    BIND_FORMAT = {'http://malc0de.com/bl/ZONES',
+                   'https://malc0de.com/bl/ZONES'}
+
+    IP_BLACKLIST = {'http://malc0de.com/bl/IP_Blacklist.txt',
+                    'https://malc0de.com/bl/IP_Blacklist.txt'}
+
+    def parse_line(self, line, report):
+        lastgenerated = None
+
+        if line.startswith('//') or len(line) == 0:
+            self.tempdata.append(line)
+            if '// Last updated' in line:
+                self.lastgenerated = line.strip('// Last updated ')
+                self.lastgenerated = dateutil.parser.parse(self.lastgenerated + 'T00:00:00+00:00').isoformat()
+
+        else:
+            event = Event(report)
```

Comment:
Here it should also be `self.new_event(report)`. Don't forget to remove the Event imports.